src/main/java/com/intel/genomicsdb/reader/GenomicsDBFeatureReader.java

```diff
@@ -63,12 +85,14 @@
     public GenomicsDBFeatureReader(final GenomicsDBExportConfiguration.ExportConfiguration exportConfiguration,
                                    final FeatureCodec<T, SOURCE> codec,
                                    final Optional<String> loaderJSONFile) throws IOException {
+	if (isGenomicsDBArray("workspacce", "array")) 
+	    throw new IllegalStateException("Not available");
```

Comment:
What's this?

Reply:
Will remove some debug stuff that crept in.

Reply:
Also refactoring GenomicsDBUtilsJni into GenomicsDBUtils and GenomicsDBUtilsJni. That way gatk can import GenomicsDBUtils for TileDB/Filesystem functionality.